dataactbroker/README.md

```diff
@@ -597,17 +597,9 @@ Example output if there are no files available:
 }
 ```
 
-#### POST "/v1/get_obligations/"
+#### GET "/v1/get_obligations/?submission_id=123"
```

Comment:
If we're already updating this, can we make it look more like the others we've updated recently? (see `check_status` for example)